$NetBSD: patch-cl,v 1.15 2018/08/19 18:32:27 wiz Exp $

- Leave gdbus-2.0/codegen sudirectory to devel/gdbus-codegen package.
- Don't compile tests as they include gdbus-codegen tests.
- Don't install completion data.
- Re-enable gdesktopappinfo on Darwin.

--- gio/Makefile.am.orig	2018-08-17 00:03:20.000000000 +0000
+++ gio/Makefile.am
@@ -1,6 +1,6 @@
 include $(top_srcdir)/glib.mk
 
-SUBDIRS = gdbus-2.0/codegen
+SUBDIRS = 
 
 if OS_UNIX
 SUBDIRS += xdgmime
@@ -246,7 +246,7 @@ platform_libadd += win32/libgiowin32.la
 platform_deps += win32/libgiowin32.la
 endif
 
-SUBDIRS += . tests
+SUBDIRS += .
 
 if HAVE_FAM
 SUBDIRS += fam
@@ -255,9 +255,7 @@ endif
 if OS_UNIX
 platform_libadd += xdgmime/libxdgmime.la
 platform_deps += xdgmime/libxdgmime.la
-if !OS_COCOA
 appinfo_headers += gdesktopappinfo.h
-endif
 
 
 unix_sources = \
@@ -396,11 +394,9 @@ portal_sources = \
 	$(NULL)
 
 if OS_UNIX
-if !OS_COCOA
 contenttype_sources += gcontenttype.c
 appinfo_sources += gdesktopappinfo.c
 endif
-endif
 
 gio_base_sources =		\
 	gappinfo.c 		\
@@ -610,9 +606,6 @@ libgio_objc_2_0_la_CPPFLAGS = $(libgio_2
 libgio_objc_2_0_la_LDFLAGS = $(libgio_2_0_la_LDFLAGS) -Wl,-framework,Foundation -Wl,-framework,AppKit
 libgio_objc_2_0_la_SOURCES = \
 	gnextstepsettingsbackend.c \
-	gosxcontenttype.c \
-	gosxappinfo.c \
-	gosxappinfo.h
 if MAC_OS_X_9
 libgio_objc_2_0_la_SOURCES += \
 	gcocoanotificationbackend.c
@@ -893,7 +886,6 @@ gdbus_LDADD = libgio-2.0.la 				\
 	$(NULL)
 
 if OS_UNIX
-if !OS_COCOA
 # ------------------------------------------------------------------------
 # gapplication(1) tool
 bin_PROGRAMS += gapplication
@@ -904,7 +896,6 @@ gapplication_LDADD = libgio-2.0.la 			\
 	$(top_builddir)/glib/libglib-2.0.la		\
 	$(NULL)
 endif
-endif
 
 completiondir = $(datadir)/bash-completion/completions
 completion_DATA = \
